src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetPackageSearchService.cs

```diff
@@ -196,6 +196,34 @@ public NuGetPackageSearchService(ServiceActivationOptions options, IServiceBroke
             return await GetPackageVersionsAsync(identity, packageSources, includePrerelease, isTransitive, projects: null, cancellationToken);
         }
 
+        public async Task<(bool?, string)> TryGetPackageReadMeAsync(PackageIdentity identity, IReadOnlyCollection<PackageSourceContextInfo> packageSources, bool includePrerelease, CancellationToken cancellationToken)
```

Comment:
In general, `Try*` methods return boolean value about the result of the operation and `out` parameter with the value. Can we use that pattern instead? I am curious to learn if there any reason why this pattern is more appropriate.

Reply:
You can't declare out parameters for an async method, https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/async#return-types